### DISCUSSION TITLE
RESET/3D CAMERA BUTTONS

### DISCUSSION THREAD

**Discussion by Lunaire333** [AmyE123/TactiKit#7] [Map Editor Feedback]
Having a 3D camera option is genius. It helps the user to visualize his project from a different angle properly and not miss any single detail or logic behind the concept of its map design.

However, having both 3D and reset camera functions on the same button with the subcategory showing when you hover over it would be better. It also depends on what else would be included as tools, but that would make the aesthetic of the toolkit better and more organized.

![image](https://github.com/AmyE123/TactiKit/assets/158181584/311d4d44-1918-4f55-b295-4604f04fc349)

**Comment by AmyE123** [comment]
I don't think I understand, do you have an example?

**Reply by Lunaire333** [reply]
![Hover](https://github.com/AmyE123/TactiKit/assets/158181584/83ce0b1e-ccac-4623-a087-628d2a205fe2)


It would be something like that?

**Comment by AmyE123** [comment]
A UI fix for this was mentioned on [@Jiunnyih-Tan](https://github.com/Jiunnyih-Tan)'s UI concept post: https://github.com/AmyE123/TactiKit/discussions/16
